src/system/mod.rs

```diff
@@ -118,25 +118,34 @@ impl System {
             .collect()
     }
 
```

Comment:
Please add comment for `dur_hint`.

Reply:
Since dur_hint is not exposed to user, I don't think it is really necessary to add comment.